### PR TITLE
[sonos] Removed dependency on 'org.apache.commons.io.IOUtils'

### PR DESCRIPTION
**Pull request by lolodomo**
Relative to #7722

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
This was pushed without a working maven buiuld (Maven build is currently not working due to download failures).

**Comment by lolodomo** [comment]
> This was pushed without a working maven buiuld (Maven build is currently not working due to download failures).

Using the offline mode (`mvn -o clean install`), the local build succeeded.

File: bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosAudioSink.java [hunk 1]
```diff
@@ -115,7 +114,11 @@ public void process(AudioStream audioStream)
                 logger.warn("We do not have any callback url, so Sonos cannot play the audio stream!");
             }
         } else {
-            IOUtils.closeQuietly(audioStream);
+            try {
+                audioStream.close();
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosAudioSink.java:118]: I once again have to jump into the discussion if we really have to close the passed `AudioStream` in `AudioSink` implementations. IIRC Chromecast and Kodi do not close it and I am not aware of a resource leak. Maybe we can find a final decision on it in https://github.com/openhab/openhab-core/issues/532.

**Comment by lolodomo** [comment]
As it is apparently not clear if the audio stream has to be closed or not, I close my change proposal until this point is clearer.

**Pull request closed without merging**